### DISCUSSION TITLE
Cannot connect Moes scene switch

### DISCUSSION THREAD

**Discussion by mlewando** [Koenkk/zigbee2mqtt#23932] [Devices]
I have a couple of Moes Star Ring switches already connected to my network, 8 switches that are connected to mains and  4 powered by batteries. I have a bunch of other devices (46 routers included). Today I tried to add one more Moes switch that runs on batteries, but after I press "permit to join" in the Zigbee2Mqtt and enter the pairing mode on the switch, nothing happens... Nothing in the logs as well...
I tried to pair some other device (Aquara door sensor) that I had handy and that worked correctly. I also tried to permit to join only the coordinator and put the new switch close to the coordinator (0cm close) without luck... I tried to connect to some router that according to the map had no other nodes connected, and also no luck, I tried with some other moes switches (I have 5 of them), none of them worked... quite unlikely that all 5 are broken.

The whole network (including already connected moes switches) is working correctly...

My setup:
Zigbee2MQTT version: 1.40.1
Coordinator type: ConBee3
Coordinator revision: 0x26530900
Frontend version: 0.7.4
zigbee-herdsman-converters version: 20.12.1
zigbee-herdsman version: 0.57.3

HA:
Core: 2024.9.1
Supervisor: 2024.08.0
Operating System: 12.3
Frontend: 20240906.0

Other (already paired) battery switches are listed in devices list with Zigbee Model TS0043

is there anything that I can do to diagnose and solve the issue?

**Comment by ustari28** [comment]
I have the same problem.

**Comment by ustari28** [comment]
I just test again and only pair with the main router, when I try with another router in the network the device doesn't pair. Maybe, is it a problem with new version of zigbee2mqtt software?

**Comment by ustari28** [comment]
After some tests, I found that is a problem with the signal range then, I added a new ZBMINIR2 with better signal range and erased and paired the devices again, and the issue was resolved in my case..
